src/script/view_model/CallingViewModel.ts

```diff
@@ -103,7 +103,7 @@ export class CallingViewModel {
     private readonly conversationState = container.resolve(ConversationState),
     readonly callState = container.resolve(CallState),
     private readonly teamState = container.resolve(TeamState),
-    core = container.resolve(Core),
+    private readonly core = container.resolve(Core),
```

Comment:
humm I'm wondering why it was like this in the first place 🤔

Reply:
good question 💁‍♂️